src/course/syllabus/application/feature/learning-outcomes.md

```diff
@@ -0,0 +1,38 @@
+## Planning
+
+- [ ] Be able to break a feature into smaller parts
+- [ ] Be confident in writing pseudocode for each part of a feature
+- [ ] Be able to design an intuitive user interface
+
+## HTML
```

Comment:
i think we can remove the html and css outcomes from here. They're not very specific/relevant to this project and have been covered in earlier projects